### DISCUSSION TITLE
Debian/ubuntu packages for v2

### DISCUSSION THREAD

**Discussion by francbartoli** [pydantic/pydantic#7756] [Pydantic V2]
Is pydantic v2 released/maintained as a linux Debian/Ubuntu package?

**Comment by cjwatson** [comment]
Debian testing and the current Ubuntu development release now both have pydantic 2.8.2, so pydantic v2 will be in both Debian 13 and Ubuntu 24.10.

Sorry this took so long; it took some time to get all the new Rust pieces lined up properly and sort out various bits of CI. (I just did the last push to get it over the line; other people did most of the work.)
